### PR TITLE
Missed markup

### PR DESCRIPTION
**Pull request by babenek**
in common - false
must be merged after https://github.com/Samsung/CredSweeper/pull/432

### CODE REVIEW

File: meta/e3377359.csv [hunk 1]
```diff
@@ -118,4 +118,3 @@ Id,FileID,Domain,RepoName,FilePath,LineStart:LineEnd,GroundTruth,WithWords,Value
 131695,6e017f34,GitHub,e3377359,data/e3377359/src/i.php,110:110,T,F,,,F,F,,,,,0,0,F,F,F,Other
 131702,6e017f34,GitHub,e3377359,data/e3377359/src/i.php,38:38,T,F,35,51,F,F,,,,,0,0,F,F,F,Password
 131703,6e017f34,GitHub,e3377359,data/e3377359/src/i.php,74:74,T,F,35,51,F,F,,,,,0,0,F,F,F,Password
-131704,87fccb9b,GitHub,e3377359,data/e3377359/src/r.php,355:355,T,F,169,185,F,F,,,,,0,0,F,F,F,Password
```
Comment (babenek) [inline meta/e3377359.csv:121]: duplicate

File: meta/f710ac3c.csv [hunk 2]
```diff
@@ -1069,3 +1069,4 @@ Id,FileID,Domain,RepoName,FilePath,LineStart:LineEnd,GroundTruth,WithWords,Value
 132526,cdecae61,GitHub,f710ac3c,data/f710ac3c/test/ck.txt,76:76,T,F,,,F,F,,,,,0.00,,F,F,F,Other
 132527,cdecae61,GitHub,f710ac3c,data/f710ac3c/test/ck.txt,78:78,T,F,,,F,F,,,,,0.00,,F,F,F,Other
 132126,eb20de75,GitHub,f710ac3c,data/f710ac3c/test/bh.py,922:922,T,F,,,F,F,,,,,0.00,,F,F,F,Other
+132673,d67e41a7,GitHub,f710ac3c,data/f710ac3c/test/cj.py,163:163,T,F,,,F,F,,,,,0.00,,F,F,F,Password
```
Comment (csh519) [inline meta/f710ac3c.csv:1072]: As I know our `MIN_VALUE_LENGTH` is 4, but it seems it's 3.
Does it detected by CredSweeper?

Comment (babenek) [reply meta/f710ac3c.csv:1072]: ``\`` is a charachter too

Comment (csh519) [reply meta/f710ac3c.csv:1072]: Okay.. I see.

**Review by csh519 — APPROVED** [review]
Thank you for the mark up.

LGTM :+1:

**Pull request merged**